specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1007 @@
+of the different API objects.
+
+## RBAC Profile
+
+In order to validate the controls described in
+[Resource Overview](#resource-overview), the following Kubernetes RBAC profile
+can be applied in a Kubernetes cluster. This Kubernetes RBAC is an illustrative
+example of the minimal profile rather than a requirement. This Role is
+sufficient to develop, deploy, and manage event routing for an application
+within a single namespace. Knative Conformance tests against "MUST", "MUST NOT",
+and "REQUIRED" conditions are expected to pass when using this profile:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: knative-developer
+rules:
+  - apiGroups: ["eventing.knative.dev"]
+    resources: ["broker", "trigger"]
+    verbs: ["get", "list", "create", "update", "patch", "delete"]
+  - apiGroups: ["messaging.knative.dev"]
+    resources: ["channel", "subscription"]
+    verbs: ["get", "list", "create", "update", "patch", "delete"]
+```
+
+In order to support resolving resources which meet the
+[Addressable](./overview.md#addressable) contract, the system controlling the
+[Trigger](#trigger-lifecycle) or [Subscription](#subscription-lifecycle) will
+need _read_ access to these resources. On Kubernetes, this is most easily
+achieved using role aggregation; on systems using Kubernetes RBAC, resources
+which wish to participate in [Addressable resolution](#addressable-resolution)
+are expected to provide the following `ClusterRole`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <component>-addressable-resolver
+  labels:
+    duck.knative.dev/addressable: "true"
+rules:
+  - apiGroups: [<resource apiGroup>]
+    resources: [<resource kind >, <resource kind>/status]
+    verbs: ["get", "list", "watch"]
+```
+
+This configuration advice does not indicate a requirement for Kubernetes RBAC
+support.
+
+<!-- TODO: define aggregated API roles
+
+Ref:
+- https://github.com/knative/specs/blob/main/specs/eventing/channel.md#aggregated-channelable-manipulator-clusterrole
+- https://github.com/knative/specs/blob/main/specs/eventing/channel.md#aggregated-addressable-resolver-clusterrole
+-
+-->
+
+## Error Signalling
+
+See [the Knative common condition guidance](../common/error-signalling.md) for
+how resource errors are signalled to the user.
+
+## Resource Lifecycle
+
+### Broker Lifecycle
+
+A Broker represents an Addressable endpoint (i.e. it MUST have a
+`status.address.url` field) which can receive, store, and forward events to
+multiple recipients based on a set of attribute filters (Triggers). Triggers
+MUST be associated with a Broker based on the `spec.broker` field on the
+Trigger; it is expected that the controller for a Broker will also control the
+associated Triggers. When the Broker's `Ready` condition is `true`, the Broker
+MUST provide a `status.address.url` which accepts all valid CloudEvents and MUST
```

Comment:
"all valid CEs" - do we support both binary and structured CEs?  If so, then ignore this.

Reply:
If so, do we test for binary?